lib/sprockets/server.rb

```diff
@@ -11,6 +11,14 @@ module Server
     # Supported HTTP request methods.
     ALLOWED_REQUEST_METHODS = ['GET', 'HEAD'].to_set.freeze
 
+    if Gem::Version.new(Rack::RELEASE) < Gem::Version.new("3")
+      X_CASCADE = "X-Cascade"
+      VARY = "Vary"
```

Comment:
```suggestion
      X_CASCADE = "X-Cascade" # :nodoc:
      VARY = "Vary" # :nodoc:
```

I don't want to maintain constants for strings just because Rack 3 is incompatible with Rack 2